docs/tutorial/08-create-libs.md

```diff
@@ -0,0 +1,161 @@
+# Step 8: Create Libs
```

Comment:
This feels out of place... what do you think about making this Step 3 where a Todo component is created in a lib?

Reply:
Yeah, it probably should be :) I tried to match the video I recorded, so folks can watch the video and the do the tutorial. Reordering the steps will require changing the video :) So I'd do those in sync.